docs/towers.md

```diff
@@ -1,62 +1,266 @@
 # The Towers
+Gooey Defence is a tower defence gameplay template. This means that one of the most important features of the module is the towers that give the genre its name. In Gooey Defence, these are freeform multiblock structures, with a minimum of three parts. Together these parts allow players to build a wide variety of towers.
 
-Towers are the multi-block structures that attack the enemies either by damaging them or affecting their stats.
-A tower is made up of at least three blocks:
-* a core block
-* an effector block
-* a targeter block
+The first part of a tower is the core. These add power to the other two block types.
```

Comment:
I'm wondering - in the wiki `core` here was a link to the sub-page on Cores. Would it make sense to make similar sub-pages in the /docs approach and come up with the right way to link pages that way? This otherwise becomes a pretty intense single page :-)

I'll mark this as a requested change, although I'm happy to just hear commentary, maybe my personal enjoyment of "little things" would push this into too-tiny-pages territory. Otherwise it is good to see some doc work happening?

Will leave my review stopped at this early point to first see about ideal next steps there.

Reply:
Yes, good point! The original towers.md had all the sections in together so it didn't occur to me to split it up, but hopefully this works.